### DISCUSSION TITLE
angular/creating-progress-spinner-in-angular/

### DISCUSSION THREAD

**Discussion by giscus** [arungudelli/angularjswiki#168] [Announcements]
# angular/creating-progress-spinner-in-angular/

We can create a Progress spinner in Angular using material design component mat-progress-spinner.

https://www.angularjswiki.com/angular/creating-progress-spinner-in-angular/

**Comment by BenediktHofer** [comment]
When I add the tag:

<app-mat-spinner-overlay overlay="true">
</app-mat-spinner-overlay>

into my html file

Web-Console tells me that I've got too much Recursion

"InternalError: too much recursion"

What should I do?

**Reply by arungudelli** [reply]
I do not think so it's related to Angular. Do you have a stackblitz demo link so that I can have a look?
